main.go

```diff
@@ -35,19 +36,27 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
+	if len(os.Args) > 1{
```

Comment:
I think it should have a whitespace before the curly bracket. Please use `go fmt` :)

Reply:
Done :)